packages/framework/src/Resources/config/services_test.yml

```diff
@@ -176,3 +176,12 @@ services:
     Shopsys\FrameworkBundle\Model\Cart\Watcher\CartWatcherFacade:
         arguments:
             - '@shopsys.shop.component.flash_message.sender.front'
+
+    Shopsys\FrameworkBundle\Model\Product\Search\ProductElasticsearchRepository:
+        arguments:
+            $indexPrefix: 'test-%env(ELASTIC_SEARCH_INDEX_PREFIX)%'
+
+    Shopsys\FrameworkBundle\Component\Elasticsearch\ElasticsearchStructureManager:
+        arguments:
+            - '%shopsys.elasticsearch.structure_dir%'
```

Comment:
can you also please check `packages/framework/tests/Unit/Component/Elasticsearch/ElasticsearchStructureManagerTest.php` whether that test is independent from the acceptance and integration tests and recreating structures?

Reply:
There is no need to change anything,  because there is already test prefix implemented.